webpack.common.js

```diff
@@ -49,16 +43,9 @@ module.exports = {
   plugins: [
     new WebpackCopyPlugin(apps),
     new HtmlWebpackPlugin({
-      template: 'index.html',
-      favicon: favicon
+      template: 'index.html'
     }),
-    new VueLoaderPlugin(),
-    new MiniCssExtractPlugin({
```

Comment:
I guess you need to bring this back in case we are loading css from ods again ;-)